src/EditorFeatures/Test2/IntelliSense/CSharpSignatureHelpCommandHandlerTests.vb

```diff
@@ -177,18 +177,17 @@ class Program
                 Await state.AssertSelectedSignatureHelpItem("void C.M(int first, int second)")
                 Assert.Equal(1, state.CurrentSignatureHelpPresenterSession.SignatureHelpItems.Count)
 
-                ' Keep the same item selected when the colon is deleted, but now both items are
-                ' available again.
+                ' Now both items are available again, and we're selecting the initial overload again.
                 state.SendBackspace()
                 Await state.AssertSignatureHelpSession()
-                Await state.AssertSelectedSignatureHelpItem("void C.M(int first, int second)")
+                Await state.AssertSelectedSignatureHelpItem("void C.M(int third)")
```

Comment:
strange... since we had previously selected ```int first, int second``` and that guy is still available, shouldn't we still have that one selected?  after all, the user didn't manually select anything themselves.  This actually seems like a regression (and i'm not sure why it happened).

Reply:
I'll look at the rest of the tests later.  but i def want to understand this one first.